reactivesocket-core/src/main/java/io/reactivesocket/internal/RemoteReceiver.java

```diff
@@ -176,6 +186,16 @@ public void cancel() {
         onError(new CancelException("Remote subscription cancelled."));
     }
 
+    private boolean isMissed() {
```

Comment:
why not just returning `subscription == null` ?
I don't see the need for the synchronized block.